includes/export/abstract-wc-csv-exporter.php

```diff
@@ -171,7 +171,7 @@ public function export() {
 	 */
 	public function send_headers() {
 		if ( function_exists( 'gc_enable' ) ) {
-			gc_enable();
+			gc_enable(); // phpcs:ignore PHPCompatibility.PHP.NewFunctions.gc_enableFound
```

Comment:
The exporter works only with PHP >= 5.3? If so, should maybe exclude the rule PHPCompatibility.PHP.NewFunctions.gc_enableFound from all files in `includes/export` (similar to https://github.com/woocommerce/woocommerce/pull/19269/files#diff-6b216d983cf0a62e9c648934c5687456R71) instead of using a inline exclude. Unfortunately, it seems that PHPCompatibily doesn't accept different supported PHP versions per directory.

Reply:
There is a function exists check so would not say this is only for 5.3, just an extra feature that is enabled when running 5.3+